test/Microsoft.AspNetCore.Mvc.WebApiCompatShimTest/project.json

```diff
@@ -26,10 +26,12 @@
         "dotnet-test-xunit": "1.0.0-dev-*"
       }
     },
-    "dnx451": {
+    "net451": {
```

Comment:
WebApiCompatShim gets low traffic for code changes. Is it ok to remove VS + DNX test support for this?